### PR TITLE
ledger documentation

### PR DESCRIPTION
**Pull request by leanthebean**
### What changed?

Added Ledger documentation in the Getting Started section

### CODE REVIEW

File: content/get-started/ledger.mdx [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+  />
+  <img 
+    src="/images/blog/ledger/7.png" 
+    style={{ maxWidth: '45%', height: 'auto' }}
+  />
+</div>
+
+### How to Receive Funds
+
+After importing the Ledger account into the Node App, you can find the public address for that wallet either by clicking on the public address below account name to save it to your clipboard, navigating to walelt details or by going to the Receive tab:
```
Comment (patnir) [inline content/get-started/ledger.mdx:94]:
```suggestion
After importing the Ledger account into the Node App, you can find the public address for that wallet either by clicking on the public address below account name to save it to your clipboard, navigating to wallet details or by going to the Receive tab:
```

File: content/get-started/ledger.mdx [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+After clicking on the 'Send Asset' button, you’ll need to review and approve the transaction on the Ledger device. If anything looks incorrect, you’ll have the chance to reject the transaction. 
+
+Iron Fish is a UTXO model, so a transaction consumes and produces new Notes (e.g. UTXOs). During the review process on the Ledger device you’ll see those Notes being consumed or created and the corresponding 'AssetID', 'Amount', 'Owner' for that note displayed: 
+
+![](/images/blog/ledger/13.png)
+
+Please note that amounts in Iron Fish are displayed without decimal points, meaning that 1 IRON will be displayed as 10000000 since Iron Fish has 8 decimal points for precision. 
+
+## Multi-sig Wallet Ledger Flow
```
Comment (patnir) [inline content/get-started/ledger.mdx:136]:
```suggestion
## Multi-signature Wallet Ledger Flow
```

File: content/get-started/ledger.mdx [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+
+### Requirements
+
+For the Multi-signer Ledger flow you’ll need the [Node App](https://ironfish.network/use/node-app) as well as the Iron Fish CLI. The easiest way to install the Iron Fish CLI is through NPM. If you already have NodeJS on your computer, you can install Iron Fish CLI by simply running the command:
```
Comment (patnir) [inline content/get-started/ledger.mdx:140]:
```suggestion
For the Multi-signature Ledger flow you’ll need the [Node App](https://ironfish.network/use/node-app) as well as the Iron Fish CLI. The easiest way to install the Iron Fish CLI is through NPM. If you already have NodeJS on your computer, you can install Iron Fish CLI by simply running the command:
```

File: content/get-started/ledger.mdx [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+
+## Single Signer Ledger Flow
+
+### Requirements
+
+You’ll need the latest version of the [Node App](https://ironfish.network/use/node-app) to see the balance, transaction history, and facilitate creating transactions for your Ledger wallet.  Please make sure to always have your Iron Fish [Node App](https://ironfish.network/use/node-app) up to date. Installing a new version will preserve all your chain and wallet history. 
+
+For your Ledger device, make sure to: 
+
+- Have [Ledger Live installed](https://support.ledger.com/article/4404389503889-zd)
```
Comment (patnir) [inline content/get-started/ledger.mdx:26]:
```suggestion
- Install [Ledger Live](https://support.ledger.com/article/4404389503889-zd)
```

File: content/get-started/ledger.mdx [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+  />
+  <img 
+    src="/images/blog/ledger/5.png" 
+    style={{ maxWidth: '45%', height: 'auto' }}
+  />
+</div>
+
+### How to View Account Balance
+
+After you’ve imported the view  key for your Ledger wallet, you should see your Ledger wallet displayed as a *view only* account on the Node App. It’ll take a few moments for your wallet to sync. After it’s synced, you should see your wallet balance displayed in the Node App.
```
Comment (patnir) [inline content/get-started/ledger.mdx:79]:
```suggestion
After you’ve imported the view  key for your Ledger wallet, you should see your Ledger wallet displayed as a *view-only* account on the Node App. It’ll take a few moments for your wallet to sync. After it’s synced, you should see your wallet balance displayed in the Node App.
```

Comment (patnir) [inline content/get-started/ledger.mdx:79]:
```suggestion
After you’ve imported the view  key for your Ledger wallet, you should see your Ledger wallet displayed as a *view only* account in the Node App. It’ll take a few moments for your wallet to sync. After it’s synced, you should see your wallet balance displayed in the Node App.
```

File: content/get-started/ledger.mdx (hunk 4 above)
Comment (NullSoldier) [reply content/get-started/ledger.mdx:26]: I would keep this simple since it's an instruction to the user

**Review by patnir — APPROVED** [review]

**Pull request merged**